### PR TITLE
feat: add local.settings.gradle.kts support

### PR DESCRIPTION
**Pull request by drymarev**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [versions]
-android = "7.2.0"
+android = "7.1.3"
```
[the remark this answers is not in the document]
Comment (drymarev) [reply gradle/libs.versions.toml:2]: To be able to use included build, AGP versions usually have to match. We can't update AGP in actual SDK sources yet due to a lint bug in 7.2.0.

**Review by tpettersen — CHANGES_REQUESTED** [review]
Hm. This is crashing on my end. EglBase is not set is what I am getting, even though you inject it with Dagger. 

```
java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{com.pexip.sdk.sample/com.pexip.sdk.sample.SampleActivity}: kotlin.UninitializedPropertyAccessException: lateinit property eglBase has not been initialized
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3676)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3936)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2341)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:233)
        at android.os.Looper.loop(Looper.java:344)
        at android.app.ActivityThread.main(ActivityThread.java:8184)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1034)
     Caused by: kotlin.UninitializedPropertyAccessException: lateinit property eglBase has not been initialized
        at com.pexip.sdk.sample.SampleActivity.getEglBase(SampleActivity.kt:32)
        at com.pexip.sdk.sample.SampleActivity.<init>(SampleActivity.kt:48)
        at java.lang.Class.newInstance(Native Method)
        at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:95)
        at androidx.core.app.CoreComponentFactory.instantiateActivity(CoreComponentFactory.java:45)
        at android.app.Instrumentation.newActivity(Instrumentation.java:1288)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3663)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3936) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2341) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:233) 
        at android.os.Looper.loop(Looper.java:344) 
        at android.app.ActivityThread.main(ActivityThread.java:8184) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1034) 

```

**Comment by drymarev** [comment]
Should be fixed in [0f451be](https://github.com/pexip/pexip-android-sdk/pull/5/commits/0f451be07c9c11a696924c7c34f9a265b9e88a0f)

**Review by tpettersen — APPROVED** [review]
👌

**Pull request merged**